grammars/xml.cson

```diff
@@ -403,3 +403,9 @@
         'name': 'punctuation.definition.comment.xml'
     'end': '--%?>'
     'name': 'comment.block.xml'
+    'patterns': [
+      {
+        'match': '-{3,}>'
```

Comment:
Is there a reason this is `{3,}` and not `{3}`?

Reply:
Well, if 3 dashes is invalid grammar, anything longer than 2 should also be invalid.

So if the end arrow was four long (---->)
{3} would only highlight -  **--->**
{3,} highlights the entire arrow which makes more sense, **---->**

Reply:
Hmm, I feel like we should only be highlighting the last three.
@Ingramz any opinions on this?